src/cljs/zetawar/views.cljs

```diff
@@ -110,12 +110,23 @@
              :width tiles/width :height tiles/height
              :xlink-href (site/prefix "/images/game/" image)}]))
 
-(defn tile [{:as view-ctx :keys [dispatch]} terrain]
+(defn tile [{:as view-ctx :keys [conn dispatch]} terrain]
   (let [{:keys [terrain/q terrain/r]} terrain]
     ^{:key (str q "," r)}
     [:g {:on-click #(dispatch [::events.ui/select-hex q r])
          :on-mouse-enter #(dispatch [::events.ui/hover-hex-enter q r])
-         :on-mouse-leave #(dispatch [::events.ui/hover-hex-leave q r])}
+         :on-mouse-leave #(dispatch [::events.ui/hover-hex-leave q r])
+         :cursor (if (or
```

Comment:
I'd suggest turning this entire 'if' into a track that's called here. That would remove some clutter from the view code and *might* improve performance a bit.

Reply:
Slight correction: I'd pull the condition part of the 'if' out and make that a track. Keeping the 'if' in the view is good as long as the condition is extracted.

Reply:
That makes sense. I decided to keep the condition in `views` because the `tile-mask` function did so. Should we move those conditions into separate tracks too?

Reply:
Yeah, go for it. Doing the same thing for tile-mask is a good idea too.

Reply:
Did you want to make that changes as part of this PR or should I go ahead and merge?

Reply:
Go ahead and merge. I'd rather do it in a separate PR.